src/executables/util_cs.c

```diff
@@ -3198,7 +3198,10 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
   log_path = utility_get_option_string_value (arg_map, APPLYINFO_COPIED_LOG_PATH_S, 0);
   if (log_path != NULL)
     {
-      log_path = realpath (log_path, log_path_buf);
+      if (realpath (log_path, log_path_buf) != NULL)
+	{
+	  log_path = log_path_buf;
+	}
     }
   if (log_path != NULL)
```

Comment:
it seems `log_path` never be NULL.
will `log_path`be verified in `la_log_page_check()` ?

Reply:
@joohok 
does `will log_path be verified ` mean verifying whether it is NULL or
whether it is real path?

Reply:
Ok